praas-app-ionic/src/components/Header/index.tsx

```diff
@@ -4,24 +4,43 @@ import {
   IonToolbar,
   IonButtons,
   IonButton,
-  IonTitle
+  IonTitle,
 } from '@ionic/react';
+import { connect } from 'react-redux';
+import { logoutUser } from '../../store/user/login';
 
-const Header: React.FC = () => {
+type Props = {
+  user: any;
+  logoutUser: () => void;
+};
+const Header: React.FC<Props> = ({ user, logoutUser }) => {
   return (
     <IonHeader>
       <IonToolbar>
         <IonTitle>PRAAS</IonTitle>
         <IonButtons slot="end">
-          <IonButton fill="outline" slot="primary" href="/signup">
-            Signup
-          </IonButton>
-          <IonButton fill="outline" href="/signin">
-            Signin
-          </IonButton>
+          {!user.login.loggedIn && (
+            <>
+              <IonButton fill="outline" slot="primary" href="/signup">
+                Signup
+              </IonButton>
+              <IonButton fill="outline" href="/signin">
+                Signin
+              </IonButton>
+            </>
+          )}
+          {user.login.loggedIn && (
```

Comment:
Does it make any difference ?

Reply:
This would be a more false safe approach I feel though debatable as well. I am not feeling it to be a big priority but if that makes code any neater, feel free to implement

Reply:
I've converted it to a ternary.